docs/rules/arrow-parens.md

```diff
@@ -60,10 +60,11 @@ if (a => 2) {
 } else {
  console.log('smaller')
 };
+// outputs bigger, not smaller as expected
 ```
 
-this is better, because condition of if is arrow function, not comparison.
-this should be like this, and you can notice it's not what you expect.
+The contents of the `if` statement is an arrow function, not a comparison.
+This should be rewritten like the following block to remove ambiguity if it was intentional.
```

Comment:
Could you restructure this sentence? It sounds that you should rewrite this block if ambiguity was intentional:-)